app/views/shared/_modal_layout.slim

```diff
@@ -0,0 +1,5 @@
+.modal-cntnr
```

Comment:
Yeah great point, I'll update.  Do you think all the other classes are appropriately named?